x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/indexlifecycle/IndexLifecycleRunner.java

```diff
@@ -158,6 +158,10 @@ public void onFailure(Exception e) {
     }
 
     private void runPolicy(IndexMetaData indexMetaData, ClusterState currentState) {
+        if (indexMetaData == null) {
```

Comment:
I think my preference would be to handle this in the caller or `runPolicy`, what do you think? (I'm okay either way)

Reply:
moveToStep is the only caller of this method, so I am also more-or-less indifferent to where this goes